### ISSUE TITLE
Add early stopping et callbacks

### ISSUE THREAD

**Issue by khufkens** [khufkens/torching_series#1]
- luz_callback_model_checkpoint() saves model weights after every epoch (or just in case of improvements, if so instructed).
- luz_callback_lr_scheduler() activates one of torch’s learning rate schedulers. Different scheduler objects exist, each following their own logic in dynamically updating the learning rate.
- luz_callback_early_stopping()
